src/backend/assets/js/session.js

```diff
@@ -1,20 +1,21 @@
 $(document).ready(function() {
+  // name is variable defined in the script in the session template file
+  var eventName = name;
```

Comment:
Thanks for the guidance :) Yes, I think that we have to keep a variable in the window. Otherwise, we won't be able to share the data between the hbs and js file. I will try using the approach you mentioned and update you. Thanks!